BE/src/main/java/kr/codesquad/sidedish/domain/Item.java

```diff
@@ -0,0 +1,79 @@
+package kr.codesquad.sidedish.domain;
+
+import org.springframework.data.annotation.Id;
+
+public class Item {
+
+    @Id
+    private final Long item_id;
+    private final Long category_id;
+    private final String item_name;
```

Comment:
java에서는 camelCase가 표준입니다. 프로젝트 전체가 snake_case로 작성되었네요. ㅜㅜ 다른 부분은 별도 추가 코멘트는 않겠습니다. 더 늦기전에 수정하는 편이 좋겠습니다.

```suggestion
public class Item {

    @Id
    private final Long itemId;
    private final Long categoryId;
    private final String itemName;
```